ros/src/computing/planning/motion/packages/waypoint_follower/nodes/mpc_follower/README.md

```diff
@@ -0,0 +1,115 @@
+|enable_path_smoothing|bool|path smoothing flag. This should be true when uses path resampling to reduce resampling noise.|true|
+|enable_yaw_recalculation|bool|recalculate yaw angle after resampling. Set true if yaw in received waypoints is noisy.|false|
+|path_filter_moving_ave_num|int|number of data points moving average filter for path smoothing|35|
+|path_smoothing_times|int|number of times of applying path smoothing filter|1|
+|curvature_smoothing_num|double|index distance of points used in curvature calculation: p(i-num), p(i), p(i+num). larger num makes less noisy values.|35|
+|steering_lpf_cutoff_hz|double| cutoff frequency of lowpass filter for steering output command [hz]|3.0|
+|admisible_position_error|double| stop vehicle when following position error is larger than this value [m].|5.0|
+|admisible_yaw_error_deg|double|stop vehicle when following yaw angle error is larger than this value [deg].|90.0|
+
+## mpc algorithm 
+
+|Name|Type|Description|Default value|
+|:---|:---|:---|:---|
+|qp_solver_type|string|QP solver option. described below in detail.|unconstraint_fast|
+|vehicle_model_type|string|vehicle model option. described below in detail.|kinematics|
+|prediction_horizon|int|total prediction step for MPC|70|
+|prediction_sampling_time|double|prediction period for one step [s]|0.1|
+|weight_lat_error|double|weight for lateral error|0.1|
+|weight_heading_error|double|weight for heading error|0.0|
+|weight_heading_error_squared_vel_coeff|double|weight for heading error * velocity|5.0|
+|weight_steering_input|double|weight for steering error (steer command - reference steer)|1.0|
+|weight_steering_input_squared_vel_coeff|double|weight for steering error (steer command - reference steer) * velocity|0.25|
+|weight_lat_jerk|double|weight for lateral jerk (steer(i) - steer(i-1)) * velocity|0.0|
+|weight_terminal_lat_error|double|terminal cost weight for lateral error|1.0|
+|weight_terminal_heading_error|double|terminal cost weight for heading error|0.1|
+|zero_ff_steer_deg|double|threshold of feedforward angle [deg]. feedforward angle smaller than this value is set to zero.|2.0|
+
+
+## vehicle
+
+|Name|Type|Description|Default value|
+|:---|:---|:---|:---|
+|wheelbase|double|wheel base length for vehicle model [m] |2.9|
+|steering_gear_ratio|double| gear ratio between steering and steering wheel. If a steering wheel angle is set in `/vehicle_status`, set this value to 1|20.0|
+|steering_tau|double|steering dynamics time constant (1d approzimation) for vehicle model [s]|0.3|
+|steer_lim_deg|double|steering angle limit for vehicle model [deg]. This is also used for QP constraint.|35.0|
+
+## QP solver type
+
+currently, the options are
+- unconstraint : use least square method to solve unconstraint QP with eigen.
+- unconstraint_fast : same as above. Faster, but lower accuracy for optimization.
+<!-- - qpoases_hotstart : use QPOASES with constrainted QP. -->
+
+## vehicle model type
```

Comment:
Please mention that Bicycle kinematics are being used and point to the reference here

Reply:
I changed readme. Please check if I got you correctly.

Reply:
Looks better. Thanks!